neurokernel/plsel.py

```diff
@@ -1831,7 +1831,7 @@ def make_index(cls, selector, names=[]):
                 names = range(max_levels)
 
             if selectors == ((),):
-                return pd.MultiIndex(levels=[[]], labels=[[]], names=names)
```

Comment:
I think it would be better to change this line to
`return pd.MultiIndex(levels=[[] for i in range(len(names))], labels=[[] for i in range(len(names))], names=names)`